### DISCUSSION TITLE
New Spacy visualizer

### DISCUSSION THREAD

**Discussion by moxley01** [explosion/spaCy#12526] [New Features & Project Ideas]
Not sure if this is the right place to post this, but I made a visualizer package recently that is designed to work with Spacy. For want of a better name, I've called it `spacysee` (https://github.com/moxley01/spacysee)

I built it because I find it useful - just posting it here in case others find it useful too.

**Comment by svlandeg** [comment]
Hi, thanks for sharing!
I gave it a quick spin and it looks neat :-) 

Maybe as a small nitpick, the example code in your [readme file](https://github.com/moxley01/spacysee/blob/main/README.md) could also `import spacy` because it's calling `spacy.load` later on, so it wouldn't work as-is. Also, note how we typically spell "spaCy" capitalizing the "C" (referring to the Cython parts of the library) and not the "s".

If you're interested, you could create a PR to the [spaCy Universe](https://spacy.io/universe)!

**Reply by moxley01** [reply]
Cool thanks, I made a PR [here](https://github.com/explosion/spaCy/pull/12568) !
